File: hello-world/python/hello-world.py
#!/usr/bin/env python

# docker run --rm -it -v .:/app python:alpine /app/hello-world.py

print('Hello World!')

num1 = 2
num2 = 3
sum = num1 + num2

print('The sum of {0} and {1} is {2}'.format(num1, num2, sum))

File: hello-world/nodejs/hello-world.js
// docker run --rm -it -v .:/app node:alpine node /app/hello-world.js

var util = require('util');

console.log('Hello World!');

num1 = 2;
num2 = 3;
sum = num1 + num2;

console.log(util.format('The sum of %d and %d is %d', num1, num2, sum));
